src/client/sandbox/upload/info-manager.js

```diff
@@ -1,18 +1,18 @@
 /*global atob, Blob, FileReader*/
-import * as Browser from '../../utils/browser';
-import * as HiddenInfo from './hidden-info';
 import NativeMethods from '../native-methods';
 import ServiceCommands from '../../../service-msg-cmd';
-import { SHADOW_UI_CLASSNAME_POSTFIX } from '../../../const';
 import Transport from '../../transport';
 import Settings from '../../settings';
 import FileListWrapper from './file-list-wrapper';
+import * as Browser from '../../utils/browser';
+import * as HiddenInfo from './hidden-info';
+import { SHADOW_UI_CLASSNAME_POSTFIX } from '../../../const';
+
+const FAKE_PATH_STRING         = 'C:\\fakepath\\';
```

Comment:
Just wondering why we have this Windows-specific code in production and does it even work on other OS?